benchmarks/link-to/active/bench.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
We should remove or fix this benchmark (it doesn't work on Ember 1.11 or above) and is current commented out (see `app/routes/index.js`).  Adding it back means that we will not be able to "run all" (since this one goes boom).